packages/mask/src/plugins/Trader/apis/coingecko/index.ts

```diff
@@ -113,7 +114,9 @@ export async function getCoinInfo(coinId: string) {
         `${COIN_GECKO_BASE_URL}/coins/${coinId}?developer_data=false&community_data=false&tickers=true`,
         { cache: Flags.trader_all_api_cached_enabled ? 'force-cache' : 'default' },
     )
-    return response.json() as Promise<CoinInfo>
+    const data = (await response.json()) as CoinInfo
+    data.links.blockchain_site = uniq(data.links.blockchain_site.map((url) => url.toLowerCase()))
```

Comment:
Move this to `getCoinTrending` in `packages/mask/src/plugins/Trader/apis/trending/index.ts`.